recipes/graphics/xorg/font/font-util.yaml

```diff
@@ -0,0 +1,38 @@
+inherit: [autotools, autoconf]
+
+metaEnvironment:
+   PKG_DESCRIPTION: "Font utilities dirs"
+   PKG_LICENSE: "MIT AND BSD-2-Clause"
+   PKG_LICENSE_FILE: "COPYING"
+   PKG_VERSION: "1.4.1"
+
+checkoutSCM:
+   scm: url
+   url: https://www.x.org/pub/individual/font/font-util-${PKG_VERSION}.tar.gz
+   digestSHA256: f029ae80cdd75d89bee7f7af61c21e07982adfb9f72344a158b99f91f77ef5ed
+   stripComponents: 1
+
+depends:
+   - graphics::xorg::proto::xproto-dev
+   - name: graphics::xorg::utils::util-macros
+     use: [tools, result]
+
+buildTools: [xorg-util-macros]
+buildScript: |
+   export ACLOCAL_PATH=${BOB_TOOL_PATHS['xorg-util-macros']}/usr/share/aclocal
```

Comment:
Is that really needed for `configure` and `make`?

Reply:
You're right. This is not necessary when using release tarballs. 

We don't have access to them in our build-environment. Instead we're using git-mirrors and have to run `autoconfreconfigure`. As this makes the source-workspace dirty we're using rsync + autoreconf in buildStep and need to set the correct `ACLOCAL_PATH`. That's why it's there...

For up-streaming the recipes I replaced the git-scm with the release tarball and removed only the rsync...